descope/api/client.go

```diff
@@ -367,76 +372,99 @@ type mgmtEndpoints struct {
 func (e *endpoints) SignInOTP() string {
 	return path.Join(e.version, e.auth.signInOTP)
 }
+
```

Comment:
Did you deliberately add all those spaces ?

Reply:
yes